### DISCUSSION TITLE
Angles FOV of Q6078-e camera

### DISCUSSION THREAD

**Discussion by Pedro-Leitek** [AxisCommunications/discussions#227] [General]
Hi,

I need to know the field of view of this camera accordingly with the zoom level. If I assume the values in datasheet to be correct, when the zoom is equal to 1 the field of view is 68.3º (4.4 focal length). But accordingly with this formula : 2tan(d/2f) the value is different (+70).
Also there is a pdf that shows the sensor size of axis cameras and I concluded that the 4K sensor (1/2.5'') has a width 6.223.

Is there any other variable that needs to be included so the the HFOV becomes 68.3? When doing the datasheets how did axis calculate the FOVs? It is very important to know this because I'm in a project where precision is critical. I need to know the HFOV and VFOV for a zoom = 2100.

Thank you

**Comment by vivekatoffice** [comment]
Related link:
[198](https://github.com/orgs/AxisCommunications/discussions/198)
